test/conformance/source/environment.cpp

```diff
@@ -205,21 +316,35 @@ PlatformEnvironment::parsePlatformOptions(int argc, char **argv) {
                        arg, "--platform=", sizeof("--platform=") - 1) == 0) {
             options.platform_name =
                 std::string(&arg[std::strlen("--platform=")]);
+        } else if (std::strncmp(arg, "--backend=", sizeof("--backend=") - 1) ==
+                   0) {
+            std::string backend_string{&arg[std::strlen("--backend=")]};
+            if (!parse_backend(backend_string)) {
+                return options;
+            }
         } else if (std::strncmp(arg, "--platforms_count=",
                                 sizeof("--platforms_count=") - 1) == 0) {
             options.platforms_count = std::strtoul(
                 &arg[std::strlen("--platforms_count=")], nullptr, 10);
         }
     }
 
-    /* If a platform was not provided using the --platform command line option,
+    /* If a platform was not provided using the --platform/--backend command line options,
      * check if environment variable is set to use as a fallback. */
     if (options.platform_name.empty()) {
         auto env_platform = ur_getenv("UR_CTS_ADAPTER_PLATFORM");
         if (env_platform.has_value()) {
             options.platform_name = env_platform.value();
         }
     }
+    if (!options.platform_backend) {
+        auto env_backend = ur_getenv("UR_CTS_BACKEND");
```

Comment:
Is `UR_CTS_BACKEND` new? I don't see it listed in the docs, should be added to the list here https://oneapi-src.github.io/unified-runtime/core/INTRO.html#environment-variables

Reply:
UR_CTS_ADAPTER_PLATFORM is not on that list either, so I assume that environment variables that are only relevant to the CTS (and not broadly to end users) shouldn't be on that list.

Reply:
Might just have been missed? I'd like to think anything that's configurable should be documented and not just left in the code as it's easily forgotten

Reply:
Done, wrote a paragraph explaining it (hopefully with no typos).